src/ansible/group_vars/all

```diff
@@ -56,3 +56,13 @@ user: {
     password: Secret123
   }
 }
+
+free_ipa_packages: {
```

Comment:
I don't really care, but AFAIK the brand is "freeipa" and it should not be split.

Reply:
changed